features/dcs_failsafe_mode.feature

```diff
@@ -19,8 +19,8 @@ Feature: dcs failsafe mode
   @dcs-failsafe
   Scenario: check one-node cluster is functioning while DCS is down
     Given DCS is down
-    When I sleep for 12 seconds
-    Then postgres0 role is the primary after 10 seconds
+    Then Response on GET http://127.0.0.1:8008/primary contains failsafe_mode_is_active after 12 seconds
```

Comment:
There are couple of more places where the similar trick could be done, but unfortunately it doesn't work with Raft because 2+ Patroni nodes are enough for quorum when `patroni_raft_controller` is paused.